pleskdistup/actions/common_checks.py

```diff
@@ -242,6 +242,54 @@ def _do_check(self) -> bool:
         return False
 
 
+class AssertOSVendorPHPUsedByWebsites(action.CheckAction):
+    min_version: version.PHPVersion
+
+    def __init__(
+            self,
+            min_version: str,
+    ):
+        self.name = "checking OS vendor PHP used by websites"
+        self.min_version = version.PHPVersion(min_version)
+        self.description = """We have detected that some domains are using the OS vendor PHP version.
+\tSwitch the following domains to {modern} or later in order to continue with the conversion process:
+\t- {domains}
+
+\tYou can achieve this by executing the following command:
+\t> plesk bin domain -u [domain] -php_handler_id plesk-php80-fastcgi
+"""
+
+    def _do_check(self) -> bool:
+        log.debug("Checking the OS vendor PHP version used by the websites")
+        if not plesk.is_plesk_database_ready():
+            log.info("Plesk database is not ready. Skipping the OS vendor PHP check.")
+            return True
+
+        os_vendor_php_handlers = ["'fpm'", "'fastcgi'"]
```

Comment:
Propose to add inner single quotes dynamically later instead of typing them here.

Reply:
Hm, seems like we could simply add them dynamically in place